python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/mpls.j2

```diff
@@ -35,3 +35,134 @@ mpls icmp fragmentation-needed tunneling
+   fast-reroute bypass tunnel optimization interval {{ fast_reroute.bypass_tunnel_optimization_interval }} seconds
+{%         endif %}
+{%     endif %}
+{%     if mpls.rsvp.srlg.enabled is arista.avd.defined(true) %}
+{%         set srlg_cli = "srlg" %}
+{%         if mpls.rsvp.srlg.strict is arista.avd.defined(true) %}
+{%             set srlg_cli = srlg_cli ~ " strict" %}
+{%         endif %}
+   {{ srlg_cli }}
+{%     endif %}
+{%     if mpls.rsvp.label_local_termination is arista.avd.defined %}
+   label local-termination {{ mpls.rsvp.label_local_termination }}
+{%     endif %}
+{%     if mpls.rsvp.preemption_method.preemption is arista.avd.defined %}
+{%         set preemption_cli = "preemption method " ~ mpls.rsvp.preemption_method.preemption %}
+{%         if mpls.rsvp.preemption_method.timer is arista.avd.defined %}
+{%             set preemption_cli = preemption_cli ~ " timer " ~ mpls.rsvp.preemption_method.timer %}
+{%         endif %}
+   {{ preemption_cli }}
+{%     endif %}
+{%     if mpls.rsvp.mtu_signaling is arista.avd.defined %}
+   mtu signaling
+{%     endif %}
+{%     if mpls.rsvp.hitless_restart.enabled is arista.avd.defined(true) %}
+   !
+   hitless-restart
+{%         if mpls.rsvp.hitless_restart.timer_recovery is arista.avd.defined %}
+      timer recovery {{ mpls.rsvp.hitless_restart.timer_recovery }} seconds
+{%         endif %}
+{%     endif %}
+{%     if mpls.rsvp.graceful_restart is arista.avd.defined %}
+{%         set graceful_restart = mpls.rsvp.graceful_restart %}
+{%         if graceful_restart.role_helper.enabled is arista.avd.defined(true) %}
+   !
+   graceful-restart role helper
+{%             if graceful_restart.role_helper.grace_period_recovery is arista.avd.defined %}
+      timer restart maximum {{ graceful_restart.role_helper.grace_period_recovery }} seconds
+{%             endif %}
+{%             if graceful_restart.role_helper.grace_period_restart is arista.avd.defined %}
+      timer recovery maximum {{ graceful_restart.role_helper.grace_period_restart }} seconds
+{%             endif %}
+{%         endif %}
+{%         if graceful_restart.role_speaker.enabled is arista.avd.defined(true) %}
+   !
+   graceful-restart role speaker
+{%             if graceful_restart.role_speaker.grace_period_recovery is arista.avd.defined %}
+      timer restart {{ graceful_restart.role_speaker.grace_period_recovery }} seconds
+{%             endif %}
+{%             if graceful_restart.role_speaker.grace_period_restart is arista.avd.defined %}
+      timer recovery {{ graceful_restart.role_speaker.grace_period_restart }} seconds
+{%             endif %}
+{%         endif %}
+{%     endif %}
+{%     if mpls.rsvp.p2mp_enabled is arista.avd.defined(true) %}
+   !
+   p2mp
+{%     elif mpls.rsvp.p2mp_enabled is arista.avd.defined(false) %}
+   !
+   p2mp
+      disabled
+{%     endif %}
+{%     if mpls.rsvp.shutdown is arista.avd.defined(true) %}
+   shutdown
```

Comment:
i think shutdown is default one, we should have no shutdown also.

Reply:
added `no shutdown` for `false` condition